nixos/tests/cjdns.nix

```diff
@@ -7,6 +7,9 @@ let
     { ... }:
     { services.cjdns.enable = true;
 
+      # Occupies port 53 otherwise
+      services.resolved.enable = false;
```

Comment:
CJDNS is actually unrelated to DNS (at least in a direct sense), this shouldn't be necesarry